From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SC26re user <@>
Date: Sat, 4 Jul 2026 19:06:30 +0200
Subject: [PATCH] drivers: sensor: lsm6dsv16x: Fence generated-style tables


diff --git a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
index 3ac7110786c..988b028d129 100644
--- a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
+++ b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
@@ -26,6 +26,7 @@ LOG_MODULE_REGISTER(LSM6DSV16X, CONFIG_SENSOR_LOG_LEVEL);
  * values taken from lsm6dsv16x_data_rate_t in hal/st module. The mode/accuracy
  * should be selected through accel-odr property in DT
  */
+/* clang-format off */
 static const float lsm6dsv16x_odr_map[3][13] = {
 			/* High Accuracy off */
 			{0.0f, 1.875f, 7.5f, 15.0f, 30.0f, 60.0f,
@@ -42,6 +43,7 @@ static const float lsm6dsv16x_odr_map[3][13] = {
 			 100.0f, 200.0f, 400.0f, 800.0f, 1600.0f,
 			 3200.0f, 6400.0f},
 		};
+/* clang-format on */
 
 static int lsm6dsv16x_freq_to_odr_val(const struct device *dev, uint16_t freq)
 {
@@ -901,6 +903,7 @@ static int lsm6dsv16x_init(const struct device *dev)
  * LSM6DSV16X_DEFINE_I2C().
  */
 
+/* clang-format off */
 #define LSM6DSV16X_DEVICE_INIT(inst)					\
 	SENSOR_DEVICE_DT_INST_DEFINE(inst,				\
 			    lsm6dsv16x_init,				\
@@ -978,3 +981,4 @@ static int lsm6dsv16x_init(const struct device *dev)
 	LSM6DSV16X_DEVICE_INIT(inst)
 
 DT_INST_FOREACH_STATUS_OKAY(LSM6DSV16X_DEFINE)
+/* clang-format on */
