From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SC26re user <@>
Date: Sat, 4 Jul 2026 19:06:57 +0200
Subject: [PATCH] drivers: sensor: lsm6dsv16x: Run clang-format


diff --git a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
index 988b028d129..26066e0ce1c 100644
--- a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
+++ b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.c
@@ -209,10 +209,8 @@ static int lsm6dsv16x_accel_range_set(const struct device *dev, int32_t range)
 	return 0;
 }
 
-static int lsm6dsv16x_accel_config(const struct device *dev,
-				enum sensor_channel chan,
-				enum sensor_attribute attr,
-				const struct sensor_value *val)
+static int lsm6dsv16x_accel_config(const struct device *dev, enum sensor_channel chan,
+				   enum sensor_attribute attr, const struct sensor_value *val)
 {
 	const struct lsm6dsv16x_config *cfg = dev->config;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
@@ -299,10 +297,8 @@ static int lsm6dsv16x_gyro_range_set(const struct device *dev, int32_t range)
 	return 0;
 }
 
-static int lsm6dsv16x_gyro_config(const struct device *dev,
-			       enum sensor_channel chan,
-			       enum sensor_attribute attr,
-			       const struct sensor_value *val)
+static int lsm6dsv16x_gyro_config(const struct device *dev, enum sensor_channel chan,
+				  enum sensor_attribute attr, const struct sensor_value *val)
 {
 	const struct lsm6dsv16x_config *cfg = dev->config;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
@@ -340,10 +336,8 @@ static int lsm6dsv16x_gyro_config(const struct device *dev,
 	return 0;
 }
 
-static int lsm6dsv16x_attr_set(const struct device *dev,
-			    enum sensor_channel chan,
-			    enum sensor_attribute attr,
-			    const struct sensor_value *val)
+static int lsm6dsv16x_attr_set(const struct device *dev, enum sensor_channel chan,
+			       enum sensor_attribute attr, const struct sensor_value *val)
 {
 #if defined(CONFIG_LSM6DSV16X_SENSORHUB)
 	struct lsm6dsv16x_data *data = dev->data;
@@ -429,8 +423,7 @@ static int lsm6dsv16x_sample_fetch_shub(const struct device *dev)
 }
 #endif /* CONFIG_LSM6DSV16X_SENSORHUB */
 
-static int lsm6dsv16x_sample_fetch(const struct device *dev,
-				enum sensor_channel chan)
+static int lsm6dsv16x_sample_fetch(const struct device *dev, enum sensor_channel chan)
 {
 #if defined(CONFIG_LSM6DSV16X_SENSORHUB)
 	struct lsm6dsv16x_data *data = dev->data;
@@ -468,20 +461,18 @@ static int lsm6dsv16x_sample_fetch(const struct device *dev,
 }
 
 static inline void lsm6dsv16x_accel_convert(struct sensor_value *val, int raw_val,
-					 uint32_t sensitivity)
+					    uint32_t sensitivity)
 {
 	int64_t dval;
 
 	/* Sensitivity is exposed in ug/LSB */
 	/* Convert to m/s^2 */
-	dval = (int64_t)(raw_val) * sensitivity;
+	dval = (int64_t)(raw_val)*sensitivity;
 	sensor_ug_to_ms2(dval, val);
 }
 
-static inline int lsm6dsv16x_accel_get_channel(enum sensor_channel chan,
-					    struct sensor_value *val,
-					    struct lsm6dsv16x_data *data,
-					    uint32_t sensitivity)
+static inline int lsm6dsv16x_accel_get_channel(enum sensor_channel chan, struct sensor_value *val,
+					       struct lsm6dsv16x_data *data, uint32_t sensitivity)
 {
 	uint8_t i;
 
@@ -507,28 +498,25 @@ static inline int lsm6dsv16x_accel_get_channel(enum sensor_channel chan,
 	return 0;
 }
 
-static int lsm6dsv16x_accel_channel_get(enum sensor_channel chan,
-				     struct sensor_value *val,
-				     struct lsm6dsv16x_data *data)
+static int lsm6dsv16x_accel_channel_get(enum sensor_channel chan, struct sensor_value *val,
+					struct lsm6dsv16x_data *data)
 {
 	return lsm6dsv16x_accel_get_channel(chan, val, data, data->acc_gain);
 }
 
 static inline void lsm6dsv16x_gyro_convert(struct sensor_value *val, int raw_val,
-					uint32_t sensitivity)
+					   uint32_t sensitivity)
 {
 	int64_t dval;
 
 	/* Sensitivity is exposed in udps/LSB */
 	/* So, calculate value in 10 udps unit and then to rad/s */
-	dval = (int64_t)(raw_val) * sensitivity / 10;
+	dval = (int64_t)(raw_val)*sensitivity / 10;
 	sensor_10udegrees_to_rad(dval, val);
 }
 
-static inline int lsm6dsv16x_gyro_get_channel(enum sensor_channel chan,
-					   struct sensor_value *val,
-					   struct lsm6dsv16x_data *data,
-					   uint32_t sensitivity)
+static inline int lsm6dsv16x_gyro_get_channel(enum sensor_channel chan, struct sensor_value *val,
+					      struct lsm6dsv16x_data *data, uint32_t sensitivity)
 {
 	uint8_t i;
 
@@ -554,16 +542,14 @@ static inline int lsm6dsv16x_gyro_get_channel(enum sensor_channel chan,
 	return 0;
 }
 
-static int lsm6dsv16x_gyro_channel_get(enum sensor_channel chan,
-				    struct sensor_value *val,
-				    struct lsm6dsv16x_data *data)
+static int lsm6dsv16x_gyro_channel_get(enum sensor_channel chan, struct sensor_value *val,
+				       struct lsm6dsv16x_data *data)
 {
 	return lsm6dsv16x_gyro_get_channel(chan, val, data, data->gyro_gain);
 }
 
 #if defined(CONFIG_LSM6DSV16X_ENABLE_TEMP)
-static void lsm6dsv16x_gyro_channel_get_temp(struct sensor_value *val,
-					  struct lsm6dsv16x_data *data)
+static void lsm6dsv16x_gyro_channel_get_temp(struct sensor_value *val, struct lsm6dsv16x_data *data)
 {
 	int32_t micro_c;
 
@@ -579,7 +565,7 @@ static void lsm6dsv16x_gyro_channel_get_temp(struct sensor_value *val,
 
 #if defined(CONFIG_LSM6DSV16X_SENSORHUB)
 static inline void lsm6dsv16x_magn_convert(struct sensor_value *val, int raw_val,
-					uint16_t sensitivity)
+					   uint16_t sensitivity)
 {
 	double dval;
 
@@ -589,9 +575,8 @@ static inline void lsm6dsv16x_magn_convert(struct sensor_value *val, int raw_val
 	val->val2 = (int32_t)dval % 1000000;
 }
 
-static inline int lsm6dsv16x_magn_get_channel(enum sensor_channel chan,
-					   struct sensor_value *val,
-					   struct lsm6dsv16x_data *data)
+static inline int lsm6dsv16x_magn_get_channel(enum sensor_channel chan, struct sensor_value *val,
+					      struct lsm6dsv16x_data *data)
 {
 	int16_t sample[3];
 	int idx;
@@ -602,13 +587,9 @@ static inline int lsm6dsv16x_magn_get_channel(enum sensor_channel chan,
 		return -ENOTSUP;
 	}
 
-
-	sample[0] = (int16_t)(data->ext_data[idx][0] |
-			     (data->ext_data[idx][1] << 8));
-	sample[1] = (int16_t)(data->ext_data[idx][2] |
-			     (data->ext_data[idx][3] << 8));
-	sample[2] = (int16_t)(data->ext_data[idx][4] |
-			     (data->ext_data[idx][5] << 8));
+	sample[0] = (int16_t)(data->ext_data[idx][0] | (data->ext_data[idx][1] << 8));
+	sample[1] = (int16_t)(data->ext_data[idx][2] | (data->ext_data[idx][3] << 8));
+	sample[2] = (int16_t)(data->ext_data[idx][4] | (data->ext_data[idx][5] << 8));
 
 	switch (chan) {
 	case SENSOR_CHAN_MAGN_X:
@@ -632,8 +613,7 @@ static inline int lsm6dsv16x_magn_get_channel(enum sensor_channel chan,
 	return 0;
 }
 
-static inline void lsm6dsv16x_hum_convert(struct sensor_value *val,
-				       struct lsm6dsv16x_data *data)
+static inline void lsm6dsv16x_hum_convert(struct sensor_value *val, struct lsm6dsv16x_data *data)
 {
 	float rh;
 	int16_t raw_val;
@@ -646,8 +626,7 @@ static inline void lsm6dsv16x_hum_convert(struct sensor_value *val,
 		return;
 	}
 
-	raw_val = (int16_t)(data->ext_data[idx][0] |
-			   (data->ext_data[idx][1] << 8));
+	raw_val = (int16_t)(data->ext_data[idx][0] | (data->ext_data[idx][1] << 8));
 
 	/* find relative humidty by linear interpolation */
 	rh = (ht->y1 - ht->y0) * raw_val + ht->x1 * ht->y0 - ht->x0 * ht->y1;
@@ -658,8 +637,7 @@ static inline void lsm6dsv16x_hum_convert(struct sensor_value *val,
 	val->val2 = rh * 1000000;
 }
 
-static inline void lsm6dsv16x_press_convert(struct sensor_value *val,
-					 struct lsm6dsv16x_data *data)
+static inline void lsm6dsv16x_press_convert(struct sensor_value *val, struct lsm6dsv16x_data *data)
 {
 	int32_t raw_val;
 	int idx;
@@ -670,19 +648,17 @@ static inline void lsm6dsv16x_press_convert(struct sensor_value *val,
 		return;
 	}
 
-	raw_val = (int32_t)(data->ext_data[idx][0] |
-			   (data->ext_data[idx][1] << 8) |
-			   (data->ext_data[idx][2] << 16));
+	raw_val = (int32_t)(data->ext_data[idx][0] | (data->ext_data[idx][1] << 8) |
+			    (data->ext_data[idx][2] << 16));
 
 	/* Pressure sensitivity is 4096 LSB/hPa */
 	/* Convert raw_val to val in kPa */
 	val->val1 = (raw_val >> 12) / 10;
-	val->val2 = (raw_val >> 12) % 10 * 100000 +
-		(((int32_t)((raw_val) & 0x0FFF) * 100000L) >> 12);
+	val->val2 =
+		(raw_val >> 12) % 10 * 100000 + (((int32_t)((raw_val) & 0x0FFF) * 100000L) >> 12);
 }
 
-static inline void lsm6dsv16x_temp_convert(struct sensor_value *val,
-					struct lsm6dsv16x_data *data)
+static inline void lsm6dsv16x_temp_convert(struct sensor_value *val, struct lsm6dsv16x_data *data)
 {
 	int16_t raw_val;
 	int idx;
@@ -693,8 +669,7 @@ static inline void lsm6dsv16x_temp_convert(struct sensor_value *val,
 		return;
 	}
 
-	raw_val = (int16_t)(data->ext_data[idx][3] |
-			   (data->ext_data[idx][4] << 8));
+	raw_val = (int16_t)(data->ext_data[idx][3] | (data->ext_data[idx][4] << 8));
 
 	/* Temperature sensitivity is 100 LSB/deg C */
 	val->val1 = raw_val / 100;
@@ -702,9 +677,8 @@ static inline void lsm6dsv16x_temp_convert(struct sensor_value *val,
 }
 #endif
 
-static int lsm6dsv16x_channel_get(const struct device *dev,
-			       enum sensor_channel chan,
-			       struct sensor_value *val)
+static int lsm6dsv16x_channel_get(const struct device *dev, enum sensor_channel chan,
+				  struct sensor_value *val)
 {
 	struct lsm6dsv16x_data *data = dev->data;
 
diff --git a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.h b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.h
index 76132f743cc..a48a9e00f51 100644
--- a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.h
+++ b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.h
@@ -26,14 +26,14 @@
 #include <zephyr/drivers/i2c.h>
 #endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c) */
 
-#define LSM6DSV16X_EN_BIT					0x01
-#define LSM6DSV16X_DIS_BIT					0x00
+#define LSM6DSV16X_EN_BIT  0x01
+#define LSM6DSV16X_DIS_BIT 0x00
 
 /* Accel sensor sensitivity grain is 61 ug/LSB */
-#define GAIN_UNIT_XL				(61LL)
+#define GAIN_UNIT_XL (61LL)
 
 /* Gyro sensor sensitivity grain is 4.375 udps/LSB */
-#define GAIN_UNIT_G				(4375LL)
+#define GAIN_UNIT_G (4375LL)
 
 struct lsm6dsv16x_config {
 	stmdev_ctx_t ctx;
@@ -67,7 +67,7 @@ union samples {
 	};
 } __aligned(2);
 
-#define LSM6DSV16X_SHUB_MAX_NUM_TARGETS			3
+#define LSM6DSV16X_SHUB_MAX_NUM_TARGETS 3
 
 struct lsm6dsv16x_data {
 	const struct device *dev;
@@ -124,14 +124,12 @@ int lsm6dsv16x_shub_init(const struct device *dev);
 int lsm6dsv16x_shub_fetch_external_devs(const struct device *dev);
 int lsm6dsv16x_shub_get_idx(const struct device *dev, enum sensor_channel type);
 int lsm6dsv16x_shub_config(const struct device *dev, enum sensor_channel chan,
-			enum sensor_attribute attr,
-			const struct sensor_value *val);
+			   enum sensor_attribute attr, const struct sensor_value *val);
 #endif /* CONFIG_LSM6DSV16X_SENSORHUB */
 
 #ifdef CONFIG_LSM6DSV16X_TRIGGER
-int lsm6dsv16x_trigger_set(const struct device *dev,
-			const struct sensor_trigger *trig,
-			sensor_trigger_handler_t handler);
+int lsm6dsv16x_trigger_set(const struct device *dev, const struct sensor_trigger *trig,
+			   sensor_trigger_handler_t handler);
 
 int lsm6dsv16x_init_interrupt(const struct device *dev);
 #endif
diff --git a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c
index 52bd7d6c0dd..7b706624ff5 100644
--- a/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c
+++ b/drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c
@@ -114,9 +114,8 @@ static int lsm6dsv16x_enable_g_int(const struct device *dev, int enable)
 /**
  * lsm6dsv16x_trigger_set - link external trigger to event data ready
  */
-int lsm6dsv16x_trigger_set(const struct device *dev,
-			  const struct sensor_trigger *trig,
-			  sensor_trigger_handler_t handler)
+int lsm6dsv16x_trigger_set(const struct device *dev, const struct sensor_trigger *trig,
+			   sensor_trigger_handler_t handler)
 {
 	const struct lsm6dsv16x_config *cfg = dev->config;
 	struct lsm6dsv16x_data *lsm6dsv16x = dev->data;
@@ -175,18 +174,15 @@ static void lsm6dsv16x_handle_interrupt(const struct device *dev)
 		if ((status.drdy_gy) && (lsm6dsv16x->handler_drdy_gyr != NULL)) {
 			lsm6dsv16x->handler_drdy_gyr(dev, lsm6dsv16x->trig_drdy_gyr);
 		}
-
 	}
 
-	gpio_pin_interrupt_configure_dt(lsm6dsv16x->drdy_gpio,
-					GPIO_INT_EDGE_TO_ACTIVE);
+	gpio_pin_interrupt_configure_dt(lsm6dsv16x->drdy_gpio, GPIO_INT_EDGE_TO_ACTIVE);
 }
 
-static void lsm6dsv16x_gpio_callback(const struct device *dev,
-				    struct gpio_callback *cb, uint32_t pins)
+static void lsm6dsv16x_gpio_callback(const struct device *dev, struct gpio_callback *cb,
+				     uint32_t pins)
 {
-	struct lsm6dsv16x_data *lsm6dsv16x =
-		CONTAINER_OF(cb, struct lsm6dsv16x_data, gpio_cb);
+	struct lsm6dsv16x_data *lsm6dsv16x = CONTAINER_OF(cb, struct lsm6dsv16x_data, gpio_cb);
 
 	ARG_UNUSED(pins);
 
@@ -217,8 +213,7 @@ static void lsm6dsv16x_thread(void *p1, void *p2, void *p3)
 #ifdef CONFIG_LSM6DSV16X_TRIGGER_GLOBAL_THREAD
 static void lsm6dsv16x_work_cb(struct k_work *work)
 {
-	struct lsm6dsv16x_data *lsm6dsv16x =
-		CONTAINER_OF(work, struct lsm6dsv16x_data, work);
+	struct lsm6dsv16x_data *lsm6dsv16x = CONTAINER_OF(work, struct lsm6dsv16x_data, work);
 
 	lsm6dsv16x_handle_interrupt(lsm6dsv16x->dev);
 }
@@ -231,9 +226,8 @@ int lsm6dsv16x_init_interrupt(const struct device *dev)
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
 	int ret;
 
-	lsm6dsv16x->drdy_gpio = (cfg->drdy_pin == 1) ?
-			(struct gpio_dt_spec *)&cfg->int1_gpio :
-			(struct gpio_dt_spec *)&cfg->int2_gpio;
+	lsm6dsv16x->drdy_gpio = (cfg->drdy_pin == 1) ? (struct gpio_dt_spec *)&cfg->int1_gpio
+						     : (struct gpio_dt_spec *)&cfg->int2_gpio;
 
 	/* setup data ready gpio interrupt (INT1 or INT2) */
 	if (!gpio_is_ready_dt(lsm6dsv16x->drdy_gpio)) {
@@ -245,10 +239,8 @@ int lsm6dsv16x_init_interrupt(const struct device *dev)
 	k_sem_init(&lsm6dsv16x->gpio_sem, 0, K_SEM_MAX_LIMIT);
 
 	k_thread_create(&lsm6dsv16x->thread, lsm6dsv16x->thread_stack,
-			CONFIG_LSM6DSV16X_THREAD_STACK_SIZE,
-			lsm6dsv16x_thread, lsm6dsv16x,
-			NULL, NULL, K_PRIO_COOP(CONFIG_LSM6DSV16X_THREAD_PRIORITY),
-			0, K_NO_WAIT);
+			CONFIG_LSM6DSV16X_THREAD_STACK_SIZE, lsm6dsv16x_thread, lsm6dsv16x, NULL,
+			NULL, K_PRIO_COOP(CONFIG_LSM6DSV16X_THREAD_PRIORITY), 0, K_NO_WAIT);
 	k_thread_name_set(&lsm6dsv16x->thread, "lsm6dsv16x");
 #elif defined(CONFIG_LSM6DSV16X_TRIGGER_GLOBAL_THREAD)
 	lsm6dsv16x->work.handler = lsm6dsv16x_work_cb;
@@ -260,8 +252,7 @@ int lsm6dsv16x_init_interrupt(const struct device *dev)
 		return ret;
 	}
 
-	gpio_init_callback(&lsm6dsv16x->gpio_cb,
-			   lsm6dsv16x_gpio_callback,
+	gpio_init_callback(&lsm6dsv16x->gpio_cb, lsm6dsv16x_gpio_callback,
 			   BIT(lsm6dsv16x->drdy_gpio->pin));
 
 	if (gpio_add_callback(lsm6dsv16x->drdy_gpio->port, &lsm6dsv16x->gpio_cb) < 0) {
@@ -269,11 +260,10 @@ int lsm6dsv16x_init_interrupt(const struct device *dev)
 		return -EIO;
 	}
 
-
 	/* set data ready mode on int1/int2 */
 	LOG_DBG("drdy_pulsed is %d", (int)cfg->drdy_pulsed);
-	lsm6dsv16x_data_ready_mode_t mode = cfg->drdy_pulsed ? LSM6DSV16X_DRDY_PULSED :
-							     LSM6DSV16X_DRDY_LATCHED;
+	lsm6dsv16x_data_ready_mode_t mode =
+		cfg->drdy_pulsed ? LSM6DSV16X_DRDY_PULSED : LSM6DSV16X_DRDY_LATCHED;
 
 	ret = lsm6dsv16x_data_ready_mode_set(ctx, mode);
 	if (ret < 0) {
@@ -281,6 +271,5 @@ int lsm6dsv16x_init_interrupt(const struct device *dev)
 		return ret;
 	}
 
-	return gpio_pin_interrupt_configure_dt(lsm6dsv16x->drdy_gpio,
-					       GPIO_INT_EDGE_TO_ACTIVE);
+	return gpio_pin_interrupt_configure_dt(lsm6dsv16x->drdy_gpio, GPIO_INT_EDGE_TO_ACTIVE);
 }
